_plugins/medium_post_display.rb

```diff
@@ -0,0 +1,23 @@
+require 'feedjira'
```

Comment:
See if you had given up you wouldn't get to learn some ruby :wink: 

Just one thing, I'm not a Ruby expert but this code is not linted. Can you learn a ruby linter on it and fix it in the same commit. If you want you can use coala too they have some [RubyBears](https://github.com/coala/coala-bears/tree/master/bears/ruby)

Reply:
I am not sure how to do this.

Reply:
> I am not sure how to do this.

How to do what? Using a single linter or running a linter bear using coala?